doc/git-scenarios-and-what-to-do-about-them.md

```diff
@@ -0,0 +1,61 @@
+# Git scenarios and what to do about them
+
+Here are the different scenarios you could find yourself in and what to do
+about them.
+
+-   1: You have a clone of a Numberscope repository.
+    -   1.A: You haven't made changes.
+        -   [Create a fork](./working-with-git-and-github#create-a-fork).
+        -   [Clone your fork](./working-with-git-and-github#clone-a-repo).
+        -   [Create a branch](./working-with-git-and-github#create-a-branch).
+        -   [Push a branch](./working-with-git-and-github#push-a-branch).
+        -   [Read about basic Git operations](./working-with-git-and-github#basic-git-operations).
```

Comment:
Everything ends with "read about basic git operations" as if that's the final step.  But I would have expected the final step to be something like "Now you are working on a dedicated branch on your own fork, congrats!  Read about about basic git operations here (link) and get to work making a cool visualizer!"

Reply:
Addressed in https://github.com/numberscope/frontscope/pull/171/commits/7a08b13c8910abad04a4afc56b2e89be979bd489.